ocha_entraid.module

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @file
+ * The ocha_entraid module file.
+ */
+
+/**
+ * Implements hook_preprocess_status_messages().
+ *
+ * If the current user is not active, ensure we do not display the EntraID
+ * login error message.
+ */
+function ocha_entraid_preprocess_status_messages(&$variables) {
+
+  $messages = $variables['message_list'];
+  if (empty($messages)) {
+    return;
+  }
```

Comment:
```suggestion
  $messages = $variables['message_list'] ?? [];
  if (empty($messages)) {
    return;
  }
```

Just to prevent a PHP warning if the message_list is not in variables.